app/src/main/java/ai/bale/theguardian/adapter/ItemViewHolder.kt

```diff
@@ -1,15 +1,67 @@
 package ai.bale.theguardian.adapter
 
+
 import android.widget.TextView
 import ai.bale.theguardian.databinding.ListItemBinding
+import ai.bale.theguardian.model.News
+import android.content.Intent
+import android.net.Uri
+import android.view.LayoutInflater
+import android.view.ViewGroup
 import android.widget.ImageView
+import androidx.core.net.toUri
 import androidx.recyclerview.widget.RecyclerView
+import coil.load
+import org.jsoup.Jsoup
+
+class ItemViewHolder(private val binding: ListItemBinding) : RecyclerView.ViewHolder(binding.root) {
+    private val itemTitle: TextView = binding.newsTitle
```

Comment:
No need to define these references. These are already defined in `ListItemBinding`